src/pykeen/losses.py

```diff
@@ -267,12 +396,15 @@ class MarginRankingLoss(PairwiseLoss):
 
     hpo_default: ClassVar[Mapping[str, Any]] = dict(
         margin=dict(type=int, low=0, high=3, q=1),
+        # todo(cthoyt): how to we specify choices? Can we directly create them from the
```

Comment:
Oh this was actually relevant for #461 where I just added some docs for this! We can come back to updating this part in that PR since it's not core to this one